shell/platform/tizen/tizen_native_window.cc

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_native_window.h"
+
+#include "flutter/shell/platform/tizen/logger.h"
+
+class TizenWl2Display {
+ public:
+  TizenWl2Display() {
+    if (!ecore_wl2_init()) {
+      LoggerE("Could not initialize ecore_wl2");
+      abort();
+      return;
+    }
+    // ecore_wl2 DISPLAY
+    wl2_display_ = ecore_wl2_display_connect(nullptr);
+    if (wl2_display_ == nullptr) {
+      LoggerE("Display not found");
+      abort();
+      return;
+    }
+    ecore_wl2_sync();
+  }
+
+  ~TizenWl2Display() {
+    if (wl2_display_) {
+      ecore_wl2_display_destroy(wl2_display_);
+      wl2_display_ = nullptr;
+    }
+    ecore_wl2_shutdown();
+  }
+  Ecore_Wl2_Display* GetHandle() { return wl2_display_; }
+
+ private:
+  Ecore_Wl2_Display* wl2_display_{nullptr};
+};
+TizenWl2Display g_tizen_wl2_display;
+
+TizenNativeEGLWindow::TizenNativeEGLWindow(
+    TizenNativeWindow* tizen_native_window, int32_t w, int32_t h) {
+  egl_window_ =
+      ecore_wl2_egl_window_create(tizen_native_window->GetWindowHandle(), w, h);
+
+  egl_display_ = eglGetDisplay((EGLNativeDisplayType)ecore_wl2_display_get(
+      g_tizen_wl2_display.GetHandle()));
+  if (egl_display_ == EGL_NO_DISPLAY) {
+    LoggerE("Could not access EGL display");
+    return;
+  }
+
+  EGLint majorVersion;
+  EGLint minorVersion;
```

Comment:
Please follow the naming convention for variables.
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
I fixed it